public/wp-content/themes/glific/page-get-started.php

```diff
@@ -56,19 +56,21 @@ class="d-block w-154 text-decoration-none bg-theme-primary text-white font-heebo
     <?php $secondary_section_data = get_field('secondary_section');
     if (!empty($secondary_section_data)) : ?>
         <div class="d-flex flex-column flex-md-row w-280 w-xl-868 w-md-750 justify-content-between mb-md-26 mx-auto">
-            <?php foreach ($secondary_section_data as $data) : ?>
-                <div class="d-flex flex-column w-xl-426 w-md-360 p-8 mb-10 mb-md-0 p-xl-10 p-md-9 bg-white get-started-content-box rounded-top-30 rounded-bottom-right-30">
-                    <h2 class="fz-18 fz-md-24 w-xl-345 w-md-310"><?php echo $data['title']; ?></h2>
-                    <?php if (!empty($data['button_url']) && !empty($data['button_text'])) : ?>
-                        <div class="d-flex flex-row justify-content-center mt-5 mt-xl-7 mt-md-8">
-                            <a href="<?php $data['button_url']; ?>" 
-                                class="d-block w-full w-md-232 text-decoration-none bg-theme-secondary text-white font-heebo-regular fz-18 px-8 px-md-10 py-4 leading-27 rounded-top-15 rounded-bottom-left-15">
-                                <?php echo $data['button_text']; ?>
-                            </a>
-                        </div>
-                    <?php endif; ?>
-                </div>
-            <?php endforeach; ?>
+            <?php foreach ($secondary_section_data as $index => $data) :
+                if ($index < 2) : ?>
```

Comment:
Why do we need to add such a check?
If it is required, can you put a comment for more clarity? The comments are not preferable every where but in such cases, it will help others to understand why we put this.